ginkgo.Jenkinsfile

```diff
@@ -52,68 +51,185 @@ pipeline {
                always {
                    sh "cd ${TESTDIR}; make clean-jenkins-precheck || true"
                }
+               failure {
+                   script {
+                       if  (!currentBuild.displayName.contains('fail')) {
+                           currentBuild.displayName = 'precheck fail\n' + currentBuild.displayName
+                       }
+                   }
+               }
             }
         }
-        stage('Boot VMs'){
+        stage ("Copy code and boot vms"){
             options {
                 timeout(time: 30, unit: 'MINUTES')
             }
+
             environment {
-                TESTDIR="${WORKSPACE}/${PROJ_PATH}/test"
+                FAILFAST=setIfLabel("ci/fail-fast", "true", "false")
+                CONTAINER_RUNTIME=setIfLabel("area/containerd", "containerd", "docker")
             }
-            steps {
-                sh 'cd ${TESTDIR}; NETNEXT=true K8S_VERSION=1.10 vagrant up --no-provision'
-                sh 'cd ${TESTDIR}; K8S_VERSION=1.14 vagrant up --no-provision'
+            parallel {
+                stage('Boot vms runtime') {
```

Comment:
We have had issues in the paste with trying to boot multiple VMs at the same time with VirtualBox - will separate workspaces for each test suite eliminate such concerns? 

I'm fine trying this for now - we can always serialize booting VMs again if needed.

Reply:
I didn't see any problems around that while testing, I think vagrant has less problems here because it's started in separate directories.